.github/workflows/ci.yaml

```diff
@@ -27,10 +27,10 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.14
+      - name: Set up Go 1.16
```

Comment:
@arugal Is this acceptable to bump up the go version?

Reply:
log support need v3,whitch need upgrade skywalking-goapi,then need bring go version to 1.16
https://github.com/apache/skywalking-goapi/blob/main/go.mod

Reply:
If CI can pass, I suggest keeping 1.14

Reply:
CI can not pass because of the go version,then upgrade to 1.16,it's passed

![image](https://user-images.githubusercontent.com/3326497/201950986-3e60be5c-763e-4f7d-842a-4f2626cf74ba.png)

Reply:
I get it.

Reply:
If 1.16 is required, could we add a line in the doc to mention this?